queue-demo-premium.tf

```diff
@@ -0,0 +1,157 @@
+  key_vault_id = "${module.vault.key_vault_id}"
+}
+
+resource "azurerm_key_vault_secret" "envelopes_queue_listen_conn_str_premium" {
+  name      = "envelopes-queue-listen-connection-string-premium"
+  value     = "${module.envelopes-queue-premium.primary_listen_connection_string}"
+  key_vault_id = "${module.vault.key_vault_id}"
+}
+
+resource "azurerm_key_vault_secret" "envelopes_queue_max_delivery_count_premium" {
+  name      = "envelopes-queue-max-delivery-count-premium"
+  value     = "${var.envelope_queue_max_delivery_count}"
+  key_vault_id = "${module.vault.key_vault_id}"
+}
+
+resource "azurerm_key_vault_secret" "processed_envelopes_queue_send_conn_str_premium" {
+  name      = "processed-envelopes-queue-send-connection-string-premium"
+  value     = "${module.processed-envelopes-queue-premium.primary_send_connection_string}"
+  key_vault_id = "${module.vault.key_vault_id}"
+}
+
+resource "azurerm_key_vault_secret" "processed_envelopes_queue_listen_conn_str_premium" {
+  name      = "processed-envelopes-queue-listen-connection-string-premium"
+  value     = "${module.processed-envelopes-queue-premium.primary_listen_connection_string}"
+  key_vault_id = "${module.vault.key_vault_id}"
+}
+
+
+resource "azurerm_key_vault_secret" "payments_queue_send_conn_str_premium" {
+  name      = "payments-queue-send-connection-string-premium"
+  value     = "${module.payments-queue-premium.primary_send_connection_string}"
+  key_vault_id = "${module.vault.key_vault_id}"
+}
+
+resource "azurerm_key_vault_secret" "payments_queue_listen_conn_str_premium" {
+  name      = "payments-queue-listen-connection-string-premium"
+  value     = "${module.payments-queue-premium.primary_listen_connection_string}"
+  key_vault_id = "${module.vault.key_vault_id}"
+}
+
+# endregion
+
+# deprecated, use `envelopes_queue_primary_listen_connection_string` instead
+output "queue_primary_listen_connection_string_premium" {
+  value = "${module.envelopes-queue-premium.primary_listen_connection_string}"
+}
+
+output "envelopes_queue_primary_listen_connection_string_premium" {
+  value = "${module.envelopes-queue-premium.primary_listen_connection_string}"
+}
+
+# deprecated, use `envelopes_queue_primary_send_connection_string` instead
+output "queue_primary_send_connection_string_premium" {
+  value = "${module.envelopes-queue-premium.primary_send_connection_string}"
+}
+
+output "envelopes_queue_primary_send_connection_string_premium" {
+  value = "${module.envelopes-queue-premium.primary_send_connection_string}"
+}
+
+output "processed_envelopes_queue_primary_listen_connection_string_premium" {
+  value = "${module.processed-envelopes-queue-premium.primary_listen_connection_string}"
+}
+
+output "processed_envelopes_queue_primary_send_connection_string_premium" {
+  value = "${module.processed-envelopes-queue-premium.primary_send_connection_string}"
+}
+
+output "envelopes_queue_max_delivery_count_premium" {
+  value = "${var.envelope_queue_max_delivery_count}"
+}
```

Comment:
are these needed?

Reply:
I can't imagine why they would be as the connection strings in particular get added to keyvault as secrets anyway